operator/v1/types_csi_cluster_driver.go

```diff
@@ -279,8 +279,35 @@ type VSphereCSIDriverConfigSpec struct {
 	// If cluster Infrastructure object has a topology, values specified in
 	// Infrastructure object will be used and modifications to topologyCategories
 	// will be rejected.
+	// +listType=set
```

Comment:
this should have failed our CRD schema checker. Why would this be a safe change to make for clients?

Reply:
Previously the checker was failing without listType, I've retried it again now and with the recent changes verification is passing without it - dropping this change.